judge/views/status.py

```diff
@@ -28,6 +36,49 @@ def status_all(request):
     })
 
 
+def status_oj(request):
+    if not request.user.is_superuser or not request.user.is_staff:
+        return HttpResponseBadRequest(_("You must be admin to view this content."), content_type='text/plain')
+
+    queryset = Submission.objects.all()
```

Comment:
I don't think we should query all the submission table, we just need the last 30 days
```py
queryset = Submission.objects.filter(date__gt=datetime.datetime.today()-datetime.timedelta(days=30))
```